### PR TITLE
Proposal for adding more information to the mitecom (and small doc update)

### PR DESCRIPTION
**Pull request by SammyRamone**
Hi,

I am currently working with the mitecom and I saw that there was no possibility to share robot positions with the team mates, so I added it.
I also added a section for communication about strategies, for example the attacking direction after a kick off. Furthermore, I added a belief value for the ball and goal information.
For the moment, I just changed this in the mitecom_data.h as I wanted to make sure, that this finds your approval. If you like it, I will implement it fully.
I also did a small update on the documentation about the orientation, please check if I did explain it right.

### CODE REVIEW

File: mitecom-data.h [hunk 1]
```diff
@@ -94,9 +94,29 @@ typedef MixedTeamStateEnum MixedTeamState;
 
 /*------------------------------------------------------------------------------------------------*/
 
+/// the playing state of the robot
+enum MixedTeamSideEnum {
+	/// robot is not doing anything or incapable of doing anything
+	SIDE_LEFT                                       = 0,
```
Comment (SGSSGene) [inline mitecom-data.h]: Wrong comment?

**Comment by SGSSGene** [comment]
Hi, first of all, sorry for replying so late. Your commit looks good, There only seems to be the wrong comments around the MixedTeamSideEnum values. If you could fix that, that be great and I'll merge it.

**Comment by SammyRamone** [comment]
Hi,
I will fix the comment and implement the rest, to make sharing of positions possible in the next days.
I wil then update the pull request.

**Comment by SammyRamone** [comment]
Hi,
I pushed the new commit.
I made a short test and it seemed to work.

**Pull request merged**